docs/README.md

```
@@ -1,32 +1,38 @@
-# Mintlify Starter Kit
+# Formbricks Documentation
 
-Click on `Use this template` to copy the Mintlify starter kit. The starter kit contains examples including
+This documentation is built using Mintlify. Here's how to run it locally and contribute.
 
-- Guide pages
-- Navigation
-- Customizations
-- API Reference pages
-- Use of popular components
+### Local Development
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)